packages/vault/src/getVaultOptions.ts

```diff
@@ -0,0 +1,17 @@
+import { VaultOptions } from '@dotcom-tool-kit/types/lib/schema/vault'
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { cosmiconfig } from 'cosmiconfig'
```

Comment:
**thought**: it doesn't feel right doing it this way. this doesn't follow the same behaviour as `cli` for loading and merging option defaults from every plugin, and having a separate implementation of this in a single plugin is almost certainly going to cause inconsistencies down the line. if the support for non-`Task` plugins for loading options via `cli` isn't working, we should fix that rather than implementing it again.